examples/SPDXXMLExample-v2.2.spdx.xml

```diff
@@ -409,23 +411,23 @@ THE SOFTWARE IS PROVIDED �AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMP
   </relationships>
   <relationships>
     <spdxElementId>SPDXRef-DOCUMENT</spdxElementId>
-    <relatedSpdxElement>DocumentRef-0:SPDXRef-ToolsElement</relatedSpdxElement>
+    <relatedSpdxElement>DocumentRef-spdx-tool-1.2:SPDXRef-ToolsElement</relatedSpdxElement>
     <relationshipType>COPY_OF</relationshipType>
   </relationships>
   <relationships>
     <spdxElementId>SPDXRef-Package</spdxElementId>
-    <relatedSpdxElement>SPDXRef-Saxon</relatedSpdxElement>
-    <relationshipType>DYNAMIC_LINK</relationshipType>
+    <relatedSpdxElement>SPDXRef-JenaLib</relatedSpdxElement>
+    <relationshipType>CONTAINS</relationshipType>
   </relationships>
   <relationships>
     <spdxElementId>SPDXRef-Package</spdxElementId>
-    <relatedSpdxElement>SPDXRef-JenaLib</relatedSpdxElement>
-    <relationshipType>CONTAINS</relationshipType>
+    <relatedSpdxElement>SPDXRef-Saxon</relatedSpdxElement>
+    <relationshipType>DYNAMIC_LINK</relationshipType>
   </relationships>
   <relationships>
     <spdxElementId>SPDXRef-JenaLib</spdxElementId>
     <relatedSpdxElement>SPDXRef-Package</relatedSpdxElement>
-    <relationshipType>DYNAMIC_LINK</relationshipType>
+    <relationshipType>NOASSERTION</relationshipType>
```

Comment:
Check the relationship here.

Reply:
The examples have been updated - this should now be resolved.